ajax/libs/react-is/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-is",
+  "version": "16.12.0",
+  "filename": "umd/react-is.production.min.js",
+  "description": "Brand checking of React Elements.",
+  "homepage": "https://reactjs.org/",
+  "keywords": [
+    "react"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/facebook/react.git",
+    "directory": "packages/react-is"
+  },
+  "npmName": "react-is",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
No `dist` directory appears to be present in the most recent package version:

```
node_modules/react-is
├── LICENSE
├── README.md
├── build-info.json
├── cjs
│   ├── react-is.development.js
│   └── react-is.production.min.js
├── index.js
├── package.json
└── umd
    ├── react-is.development.js
    └── react-is.production.min.js
```

Reply:
It was copied from the react library setup here in the project, the react package has the same structure when installed.
I have removed the dist file path.